containers/podman-desktop
# website: use Montserrat font as default font

### What does this PR do?
Use Montserrat font for the website renderring

### Screenshot/screencast of this PR

![image](https://user-images.githubusercontent.com/436777/197513462-0bba50ba-c637-491f-b892-32892836206b.png)


### What issues does this PR fix or reference?

https://github.com/containers/podman-desktop/pull/675

### How to test this PR?

Check rendering

Change-Id: I762531176c41c50eabdd9ebe10f8b1016ac6f9cd
Signed-off-by: Florent Benoit <fbenoit@redhat.com>
